core/src/components/input-otp/input-otp.ios.scss

```diff
@@ -0,0 +1,20 @@
+@import "./input-otp";
+@import "../../themes/ionic.globals.ios";
+
+// iOS Input OTP
+// --------------------------------------------------
+
+:host {
+  --border-width: #{$hairlines-width};
+}
+
+:host(.has-focus) .native-input:focus {
+  --border-width: 1px;
+}
```

Comment:
These borders will only be visible on devices that support hairline (half pixel) border widths